### PR TITLE
adding ansible remediation for auditing finit module with rhel7

### PR DESCRIPTION
**Pull request by aghassemlouei**
#### Description:

- Created Ansible remediation/fix content tasks based on template_ANSIBLE_audit_rules template to add relevant rules into /etc/audit/audit.rules and /etc/audit/rules.d/ as per DISA STIG RHEL-07-030821

#### Rationale:

- No previous Ansible remediation/fix content existed

- Adds Ansible remediation for #3466

### CODE REVIEW

**Comment by yuumasato** [comment]
@openscap-ci ok to test

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    path: '{{ all_files[0] }}'
+    line: '-a always,exit -F arch=b32 -S finit_module -k module-change'
+    state: present
+    create: true
+  tags:
+    @ANSIBLE_TAGS@
+  @ANSIBLE_ENSURE_PLATFORM@
+  - audit_rules_privileged_commands_finit_module
+  - NIST-800-53-AU-12(c)
+  - DISA-STIG-RHEL-07-030821
```
[the remark this answers is not in the document]
Comment (aghassemlouei) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml]: Went ahead and removed the tags as per your comment in f97bc6e34ee5b60c5ce5117684cb0584ebd55ae6. That's good to know for the future.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- name: Inserts/replaces the finit_module rule in rules.d on x86_64
+  lineinfile:
+    path: '{{ all_files[0] }}'
+    line: '-a always,exit -F arch=b64 -S finit_module -k module-change'
+    state: present
+    create: true
+  when: audit_arch == 'b64' and @ANSIBLE_PLATFORM_CONDITION@
```
[the remark this answers is not in the document]
Comment (aghassemlouei) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml]: Apologies for that, went ahead and added it back in via 5bd18459837cee3b2ccfc8c7a6890c472fe39392.

**Comment by aghassemlouei** [comment]
Requested changes have been made, apologies for hastily resolving the conversations rather than waiting for you to resolve them.

**Comment by yuumasato** [comment]
@aghassemlouei No need for apologies, thank you for the contribution.

If your are interested, as next step, consider making a template out of this remediation.
With templating mechanism, this remediation can be quickly extended to other syscalls, such as `init`, `create`, `delete`.

**Pull request merged**